modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/macros/FrontendDataSetAdmin.macro

```diff
@@ -294,4 +310,10 @@ definition {
 			value1 = ${key_type});
 	}
 
+	macro uncheckFields {
+		Uncheck.uncheckNotVisible(
+			checkboxName = ${checkboxName},
+			locator1 = "Checkbox#ANY_CHECKBOX");
+	}
```

Comment:
Do you know if any other macro exists to do this? This seems to be really general to not exist...

Reply:
Try to use an already existing macro to uncheck checkboxes. Apart from that, LGTM :)

Reply:
Yes, you are right. We do have a macro that uncheck checkboxes (AppBuilderAdmin.uncheckCheckbox). I'll fix that and push the changes. Thanks!